aws/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: kyverno
-version: v1.1.0-14-aws
-appVersion: v1.5.1
+version: v1.1.0-15-aws
+appVersion: v1.7.5
```

Comment:
@patelrit should we use the version as `v1.7.5-n4kbuild.1` instead of just `v1.7.5` similar as nirmata n4k charts ?

Reply:
We should just use the Kyverno version here. Not sure why N4K charts are using the image build version. Please fix

Reply:
fixed, using kyverno version here (1.8.1)